### DISCUSSION TITLE
sync with AdGuard?

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uAssets#21031] [Ideas] [answered]
EasyList Hebrew has `||doubleclick.net^$xhr,redirect-rule=nooptext,domain=13news.co.il|13tv.co.il`, for `www3.doubleclick.net` if I recall correctly
AdGuard has ( https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/BaseFilter/sections/antiadblock.txt ) : 
```
! Common rules for 'www3.doubleclick.net' bait
://www3.doubleclick.net^|$xmlhttprequest,redirect=noopjson
```
Could we add `://www3.doubleclick.net^|$xhr,redirect-rule=noop.txt`?

**Comment by Yuki2718** [comment, marked as answer]
We already have `||doubleclick.net^$xhr,redirect=noop.txt`.
